app/src/main/res/values-sw600dp-land/dimens.xml

```diff
@@ -410,5 +413,4 @@
 
   <!-- AudioLanguageFragment -->
   <dimen name="audio_language_recycler_view_padding_bottom">132dp</dimen>
-
```

Comment:
please remove this newline as it is unnecessary changes

Reply:
i think all the extra newlines are removed. can you point out the line number.